src/components/content/rss/rss-render-options.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  documentToHtmlString,
+  Options,
+} from '@contentful/rich-text-html-renderer';
+import { BLOCKS } from '@contentful/rich-text-types';
+import { ContentfulCustomModel } from '../../../types/contentful.types';
+import { Converter } from 'showdown';
+
+enum EmbeddedAssetType {
+  IMAGE = 'image',
+  VIDEO = 'video',
+}
+
+const converter: Converter = new Converter();
+const markdownToHyperText = (markdown: string) => converter.makeHtml(markdown);
+
+export const rssRenderOptions: Partial<Options> = {
```

Comment:
I'm not so much into the rendering topic, can you explain why we need a separate renderer for the content blocks? Also add the explanation as comment for devs that need to work on that in the future.

Is there some way of sharing code with the rich-text renderer? How can we make sure that developers don't forget that there are now 2 places which need to be edited when a change in Contentful happens (one idea: change the file structure and make sure both files are very close to each other)?

Reply:
We introduced a dedicated renderer for the RSS XML generation, as the rich-text renderer returns custom styled components as JSX Elements as opposed to the HTML string that the Gatsby feed plugin requires. I've tried reusing the code from `rich-text.tsx` by employing `ReactDOMServer.renderToStaticMarkup(...)`. However, this lead to build problems that I haven’t been able to resolve yet.